Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -47,7 +47,7 @@ bool RenderOpenGL::InitializeFullscreen() {
     return 0;
 }
 void RenderOpenGL::CreateZBuffer() {
-    pActiveZBuffer = (int *)malloc(0x12C000);  // 640 * 480 * 4
+    pActiveZBuffer = (int *)malloc(640 * 480 * sizeof(int));  // 640 * 480 * 4
     memset32(
```

Comment:
640 * 480 * 4 != 640 * 480 * sizeof(int)

Reply:
Do you mean on x64 / other platform? Again the point is to use sizeof instead of static numbers. If there's a problem with pActiveZBuffer != 0x12C000 because of sizeof(int) not being 4 is another matter.